skimage/exposure/hdr.py

```diff
@@ -0,0 +1,326 @@
+import numpy as np
+
+
+def get_radiance(ims, exp, radiance_map):
+    """
+    Return the radiance for a series of images based upon a camera response 
+    function
+
+    Parameters
+    ----------
+    ims : list
+          list of images in the for of numpy arrays. Either mono or colour
+          (RGB, MxNx3)
+
+    exp : numpy 1D array
+          array of exposure times in seconds
+
+    radiacenMap : numpy array
+                  array mapping the counts to radiance
+
+    Returns
+    ----------
+    hdr : numpy array
+          resulting image with radiance values
+    """
+
+    den = np.ones(ims[0].shape)
+    num = np.zeros(ims[0].shape)
+    wf = np.vectorized(weight_func)
+    for idx, im in enumerate(ims):
+        gij = im.copy()
+        # For colour
+        if im.ndim == 3:
+            for ii in range(im.shape[2]):
+                gij[:, :, ii] = radiance_map[im[:, :, ii] + 1, ii]
+        else:
+            gij[:, :] = radiance_map[im[:, :, ii] + 1]
+        gij = gij - np.log(exp[idx])
+
+        wij = wf(zij)
+        num = num + wij * gij
+        den = den + wij
+
+    return num / den
+
+
+def make_hdr(ims, exp, radiance_map, depth=16):
+    """
+    Compute the HDR image from a series of images and a racianceMap
+    Based on:
+    Debevec, P. E., & Malik, J. (1997). SIGGRAPH 97 Conf. Proc., August, 3-8.
+    doi:10.1145/258734.258884
```

Comment:
below the section returns, add a section References and put

Debevec, P. E., & Malik, J. (1997). SIGGRAPH 97 Conf. Proc., August, 3-8.
Doi:10.1145/258734.258884